gis/population/population/model.py

```diff
@@ -9,6 +10,8 @@
 
 from .space import UgandaArea
 
+script_directory = os.path.dirname(os.path.abspath(__file__))
```

Comment:
It's better to use `pathlib.Path` here so that later you can do `script_directory / stuff`.

Reply:
`script_directory = Path(__file__).resolve().parent`.

Reply:
Thanks, I looked into it and it indeed is way cleaner!

I used `Path(__file__).resolve().parent` to get the directory of the script.

Implemented in `bcc44b7`.